### PR TITLE
Apply changes for MyBatis 3.3.0 #178

### PR DESCRIPTION
**Pull request by kazuki43zoo**
* defaultFetchSize
* proxyFactory

Please review #178 .

### CODE REVIEW

File: infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml [hunk 1]
```diff
@@ -9,10 +9,11 @@
         <setting name="mapUnderscoreToCamelCase" value="true" />
         <setting name="lazyLoadingEnabled" value="true" />
         <setting name="aggressiveLazyLoading" value="false" />
+        <setting name="defaultFetchSize" value="100" />
 <!--
         <setting name="defaultExecutorType" value="REUSE" />
         <setting name="jdbcTypeForNull" value="NULL" />
-        <setting name="proxyFactory" value="JAVASSIST" />
+        <setting name="proxyFactory" value="CGLIB" />
```
Comment (making) [inline infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml]: - [ ] Is this change really needed? In my understanding, Java Assist should be usually used as the proxy factory. This line seems that we are recommending cglib.

Comment (btshimizukza) [inline infra/mybatis3/projectName-domain/src/main/resources/META-INF/mybatis/mybatis-config.xml]: @making i agree your opinion.

**Comment by btshimizukza** [comment]
@making I've reflected your review comment. Please review again.
